book/fault_proofs/quick_start.md

```diff
@@ -50,14 +50,16 @@ Save the output addresses, particularly the `FACTORY_ADDRESS` output as "Factory
 ## Step 2: Run the Proposer
 
 1. Create a `.env.proposer` file in the project root directory:
+
     ```env
     # Required Configuration
     L1_RPC=<YOUR_L1_RPC_URL>
     L2_RPC=<YOUR_L2_RPC_URL>
     FACTORY_ADDRESS=<FACTORY_ADDRESS_FROM_DEPLOYMENT>
     GAME_TYPE=42
     PRIVATE_KEY=<YOUR_PRIVATE_KEY>
-    NETWORK_PRIVATE_KEY=0x0000000000000000000000000000000000000000000000000000000000000001
+    NETWORK_PRIVATE_KEY=0x0000000000000000000000000000000000000000000000000000000000000001 # Dummy network private key for mock mode
+    MOCK_MODE=true # Set to true for mock mode
```

Comment:
This is no longer necessary (I merged https://github.com/succinctlabs/op-succinct/pull/485 to use this by default). Can you also make this change in the validity docs.